### ISSUE TITLE
[ragel] support for command line options in ragel files

### ISSUE THREAD

**Issue by ehdtee** [ehdtee/colm-suite#12]
This has been requested in the past. Requested again by Denis Naumov.

%%option name = value;

There are some sticky issues involved ... how to parse the host language to discover what language we are to parse?

**Comment by ehdtee** [comment]
We are no longer taking the host language from options so we don't need to worry about host language options. Could even skip those if we wanted.

**Comment by olesenm** [comment]
Yes, it would be nice to define `-G2` for the machine directly within the source code.
